mqtt_exporter.py

```diff
@@ -455,10 +455,23 @@ def clear(self):
 
     def update(self, label_values, value):
         child = self.metric.labels(*label_values)
+
+        # Check if value is a string that looks like a JSON object
+        if isinstance(value, str) and value.startswith('{') and value.endswith('}'):
+            try:
+                parsed_value = json.loads(value)
+                if 'value' in parsed_value:
+                    value = float(parsed_value['value'])
+                else:
+                    logging.warning(f"JSON object does not contain 'value' field: {value}")
+            except json.JSONDecodeError:
+                logging.warning(f"Failed to parse value as JSON: {value}")
```

Comment:
This must not be a warning, as most metrics that were handled with mqtt_exporter until now, were handled as strings be intention.